feat:add shisui client page. (#34)
ethereum/portal-website: pages/clients.mdx

# Portal Network clients

> There are four Portal Network clients being developed in parallel: Trin, Ultralight, Fluffy and Shisui.

Reviewer: ```suggestion
There are multiple Portal Network clients being developed in parallel.  Some of the most actively developed clients are: Trin, Ultralight, Fluffy and Shisui.
```

## Why build four clients?

> Client diversity is a major source of strength for Ethereum. It means that a bug, or an exploit in one client doesn't bring down the entire network, because the chance that the same problem is replicated in multiple independently developed clients is small. For Ethereum execution and consensus clients, a healthy diversity of clients has been challenging to achieve. Part of the reason for this is because there has long been a "main" execution client and others created later, but the network effects of the original client have been very difficult to overcome. The Portal Network developers are trying to frontrun this problem by having four clients right from the start.

Reviewer: ```suggestion
Client diversity is a major source of strength for Ethereum. It means that a bug, or an exploit in one client doesn't bring down the entire network, because the chance that the same problem is replicated in multiple independently developed clients is small. For Ethereum execution and consensus clients, a healthy diversity of clients has been challenging to achieve. Part of the reason for this is because there has long been a "main" execution client and others created later, but the network effects of the original client have been very difficult to overcome. The Portal Network developers are trying to frontrun this problem by having multiple independent clients right from the start.
```

## Which client should I use?

> Whichever you like! However, you should know that all four clients are currently in development and have not had a production release. The whole network is currently experimental and is currently missing some data. That said, you are welcome to become an early adopter! pick a client, download and install, and start experimenting with us!

Reviewer: ```suggestion
Whichever you like! However, you should know that all clients that are currently in development and have not had a production release. The whole network is currently experimental and is currently missing some data. That said, you are welcome to become an early adopter! pick a client, download and install, and start experimenting with us!
```
